### PR TITLE
#20432 Update bundle_ko.properties

### LINKED ISSUE #20432: Typo report
### Description

Report from one of our users:

> I found Korea version's typo
> File-refresh(F5) 새로로침 -> 새로고침

typo_report_DBeaver

### DBeaver Version

23.1.1

### Operating System

_No response_

### Database and driver

_No response_

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by iraidaa**
dbeaver/dbeaver#20432 One of our users reported a typo in File -> Refresh in Korean version. I changed it to the correct spelling.

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties [hunk 1]
```diff
@@ -1,34 +1,21 @@
...
-pref.page.name.navigator              = \uD0D0\uC0C9
-pref.page.name.editors                = \uD3B8\uC9D1\uAE30
-pref.page.name.query.manager          = \uCFFC\uB9AC \uAD00\uB9AC\uC790
-pref.page.name.resources              = \uB9AC\uC18C\uC2A4
-pref.page.name.errorHandle            = \uC5D0\uB7EC \uCC98\uB9AC
-pref.page.name.errorLogs              = \uC5D0\uB7EC \uB85C\uADF8
 
 activity.hideE4Activity.name = Menu And Toolbar Hiding Activity
 
+bindingcontext.name.bindingView      = In Binding View
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties]: It doesn't make sense to keep English string in localizations. Just delete them

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties [hunk 2]
```diff
@@ -1,34 +1,16 @@
...
-
 aboutText = DBeaver - \uD1B5\uD569 \uB370\uC774\uD130\uBCA0\uC774\uC2A4 \uAD00\uB9AC\uD234
-view.progress.title  = \uBC31\uADF8\uB77C\uC6B4\uB4DC \uC791\uC5C5
-view.search.title    = \uAC80\uC0C9
-view.templates.title = \uD15C\uD50C\uB9BF
 
-menu.edit       = \uD3B8\uC9D1
-menu.navigate   = Navigate
-menu.navigation = \uC774\uB3D9
+activity.hideE4Activity.name = Menu And Toolbar Hiding Activity
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties]:
```suggestion

```

File: plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties [hunk 3]
```diff
@@ -50,14 +32,14 @@ command.org.jkiss.dbeaver.core.file.new.name                         = \uC0C8\uB
...
 command.org.jkiss.dbeaver.core.file.properties.name                  = \uC124\uC815
 command.org.jkiss.dbeaver.core.file.property.name                    = \uC124\uC815
-command.org.jkiss.dbeaver.core.file.refresh.name                     = \uC0C8\uB85C\uB85C\uCE68
+command.org.jkiss.dbeaver.core.file.refresh.name                     = \uC0C8\uB85C\uACE0\uCE68
 command.org.jkiss.dbeaver.core.file.rename.name                      = \uC774\uB984 \uBC14\uAFB8\uAE30
 command.org.jkiss.dbeaver.core.file.save.name                        = \uC800\uC7A5
 command.org.jkiss.dbeaver.core.file.saveas.name                      = \uB2E4\uB978 \uC774\uB984\uC73C\uB85C \uC800\uC7A5
 #command.org.jkiss.dbeaver.core.navigator.backwardHistory.name        = Backward History
 command.org.jkiss.dbeaver.core.navigator.bookmark.add.name           = \uBD81\uB9C8\uD06C \uCD94\uAC00
 #command.org.jkiss.dbeaver.core.navigator.forwardHistory.name         = Forward History
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties]:
```suggestion

```

File: plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties [hunk 4]
```diff
@@ -50,14 +32,14 @@ command.org.jkiss.dbeaver.core.file.new.name                         = \uC0C8\uB
 command.org.jkiss.dbeaver.core.file.print.name                       = \uD504\uB9B0\uD2B8
 command.org.jkiss.dbeaver.core.file.properties.name                  = \uC124\uC815
 command.org.jkiss.dbeaver.core.file.property.name                    = \uC124\uC815
-command.org.jkiss.dbeaver.core.file.refresh.name                     = \uC0C8\uB85C\uB85C\uCE68
+command.org.jkiss.dbeaver.core.file.refresh.name                     = \uC0C8\uB85C\uACE0\uCE68
 command.org.jkiss.dbeaver.core.file.rename.name                      = \uC774\uB984 \uBC14\uAFB8\uAE30
 command.org.jkiss.dbeaver.core.file.save.name                        = \uC800\uC7A5
 command.org.jkiss.dbeaver.core.file.saveas.name                      = \uB2E4\uB978 \uC774\uB984\uC73C\uB85C \uC800\uC7A5
 #command.org.jkiss.dbeaver.core.navigator.backwardHistory.name        = Backward History
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties]:
```suggestion

```

File: plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties [hunk 5]
```diff
@@ -1,34 +1,16 @@
...
-pref.page.name.navigator              = \uD0D0\uC0C9
-pref.page.name.editors                = \uD3B8\uC9D1\uAE30
-pref.page.name.query.manager          = \uCFFC\uB9AC \uAD00\uB9AC\uC790
-pref.page.name.resources              = \uB9AC\uC18C\uC2A4
-pref.page.name.errorHandle            = \uC5D0\uB7EC \uCC98\uB9AC
-pref.page.name.errorLogs              = \uC5D0\uB7EC \uB85C\uADF8
 
-activity.hideE4Activity.name = Menu And Toolbar Hiding Activity
 
+#trimmedwindow.label.dbeaver = DBeaver
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/OSGI-INF/l10n/bundle_ko.properties]:
```suggestion

```

**Review by E1izabeth — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Pull request merged**